packages/app/src/specs/SpecsListHoverCell.cy.tsx

```diff
@@ -0,0 +1,61 @@
+import SpecsListHoverCell from './SpecsListHoverCell.vue'
+
+const contentSelector = '[data-testid=content]'
+const hoverSelector = '[data-testid=hover]'
+
+describe('<SpecsListHoverCell/>', () => {
+  it('should show hover when hover enabled', () => {
+    cy.mount(() => (
+      <SpecsListHoverCell
+        isHoverDisabled={false}
+        // @ts-ignore - doesn't know about vSlots
+        vSlots={{
+          content: () => <div data-testid="content">content</div>,
+          hover: () => <div data-testid="hover">Hover</div>,
+        }}
+      />
+    )).get(contentSelector)
+    .should('be.visible')
+    .get(hoverSelector)
+    .should('not.exist')
+    .log('hover content to show hover slot')
+    .get(contentSelector)
+    .trigger('mouseenter')
+    .should('not.exist')
+    .get(hoverSelector)
+    .should('be.visible')
+    .log('hover slot should stay visible if hovering the hover slot')
```

Comment:
Oh, this is neat, cool way to document your test.

Reply:
👍